templates/keras_nlp/index.md

```diff
@@ -39,64 +43,72 @@ check out our
 ---
 ## Installation
 
-KerasNLP requires **Python 3.7+** and **TensorFlow 2.9+**.
-
-Install the latest release:
+To install the latest official release:
 
 ```
 pip install keras-nlp --upgrade
 ```
 
-You can check out release notes and versions on our
-[releases page](https://github.com/keras-team/keras-nlp/releases).
+To install the latest unreleased changes to the library, we recommend using
+pip to install directly from the master branch on github:
 
-KerasNLP is currently in pre-release (0.y.z) development. Until version 1.0, we
-may break compatibility at any time and APIs should not be considered stable.
+```
+pip install git+https://github.com/keras-team/keras-nlp.git --upgrade
+```
 
----
-## Quick introduction
+## Quickstart
 
-The following snippet will tokenize some text, build a tiny transformer, and
-train a single batch.
+Fine-tune BERT on a small sentiment analysis task using the 
+[`keras_nlp.models`](/api/keras_nlp/models/) API:
 
 ```python
 import keras_nlp
-import tensorflow as tf
 from tensorflow import keras
+import tensorflow_datasets as tfds
+
+# Use mixed precision for optimal performance
+keras.mixed_precision.set_global_policy('mixed_float16')
```

Comment:
Not sure we should enable mixed precision in a starter code thing. It's orthogonal to everything else. In a `/examples/` code example (configured to run on GPU in Colab) it's more clearly useful.